data/spineopt_template.json

```diff
@@ -115,17 +117,22 @@
         ["model", "write_lodf_file", false, "boolean_value_list", "A boolean flag for whether the LODF values should be written to a results file."],
         ["model", "write_mps_file", null, "write_mps_file_list", "A selector for writing an .mps file of the model."],
         ["model", "write_ptdf_file", false, "boolean_value_list", "A boolean flag for whether the LODF values should be written to a results file."],
+        ["model","bigM", null, null, "Sufficiently large number used for linearization bilinear terms, e.g. to enforce bidirectional flow for gas pipielines"],
```

Comment:
our parameter names are snake_case, would you consider renaming this to `big_m`?